CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+ComplianceAsCode Contribution Guidelines
+===
+
+We aim to create a welcoming environment for all contributors regardless of their age, gender, experience, nationality, or employer. However, to foster smooth collaboration we are asking contributors to adhere to the following rules.
+
+Issues and Pull Requests
+---
+
+Pull requests can’t be merged by the author, a review from another person is needed. If anybody raises material objections to the pull request, those objections should be addressed before the PR merge.
+
+If a pull request has received feedback from reviewers, and the original author hasn’t touched it for more than 14 days, the PR may be closed (it is still possible to re-open it if the author catches up later).
+
+Pull requests may be closed immediately only if the change would render the build system, a profile, or a tool broken, and, at the same time, there is no way to improve the PR. Justification and an extensive explanation (or a link to a previous case of such a change) should always be provided as a part of the feedback.
+
+Pull requests should pass continuous integration tests. If the CI run ends with an error, the merge can only be approved by an independent reviewer who has the knowledge of the CI. A waiver justification message is required to be a part of the PR.
+
+Both issues and pull requests should contain a description that clarifies the matter to a degree that a regular contributor can understand it. Issues and PRs are not a personal TODO list, they are problems and solutions descriptions for the whole community to acknowledge and act upon.
```

Comment:
```suggestion
Issues and PRs are not a personal TODO list, they are problems and solutions descriptions for the whole community to acknowledge and act upon.
```

This sentence needs to be removed. Issues can very much be a contributors TODO list for a PR.

Reply:
I think it needs rewording, instead. I think the point isn't that the issues can't look list a list of TODO items, that's fine. The problem is different. I think we actually don't want to have issues that only their author can understand. And we don't want issues that don't contain enough information to be actionable. Instead, we want issues that contributors can easily take and resolve.

Reply:
Yes, the key word is "personal". Maybe this one would be better:
> Both issues and pull requests should contain a description that clarifies the matter to a degree that a regular contributor can understand it. Please use Issues and PRs to describe problems (and possible solutions) for the whole community to acknowledge and act upon.

What to do with things like #5525 is another problem. On one hand they are part of a GH "project" and, as far as I can see, it was an attempt to build some kind of contribution pipeline. On the other hands it looks abandoned, and also its implementation sucks a bit because the project litters Issues list to the level of being non-browsable by humans.

Reply:
yes, these are inhuman